dvc/repo/get.py

```diff
@@ -66,31 +64,40 @@ def get(url, path, out=None, rev=None):
             # the same cache file might be used a few times in a directory.
             repo.cache.local.cache_types = ["reflink", "hardlink", "copy"]
 
-            output = None
-            output_error = None
-
             try:
                 output = repo.find_out_by_relpath(path)
-            except OutputNotFoundError as ex:
-                output_error = ex
-
-            is_git_file = output_error and not os.path.isabs(path)
-            is_not_cached = output and not output.use_cache
+            except OutputNotFoundError:
+                output = None
```

Comment:
It is better to save this exc and then provide it as a cause with the execption that is raised later, same as we do in repo.open

Reply:
That cause would be a lie anyway. We don't know whether user meant git file path.